content/scalability/docs/workloads.md

```diff
@@ -0,0 +1,87 @@
+# Workloads
+
+Workloads have an impact on how large your cluster can scale. Workloads that use the Kubernetes APIs heavily will limit the total amount of workloads you can have in a single cluster, but there are some defaults you can change to help reduce the load.
+
+Workloads in a Kubernetes cluster have access to features that integrate with the Kubernetes API (e.g. Secrets and ServiceAccounts), but these features are not always required and should be disabled if they’re not being used. Limiting workload access and dependence on the Kubernetes control plane will increase the number of workloads you can run in the cluster and improve the security of your clusters by removing unnecessary access to workloads and implementing least privilege practices. Please read the [security best practices](https://aws.github.io/aws-eks-best-practices/security/docs/) for more information.
+
+## Use IPv6 for pod networking
+
+You cannot transition a VPC from IPv4 to IPv6 so enabling IPv6 before provisioning a cluster is important. If you enable IPv6 in a VPC it does not mean you have to use it and if your pods and services use IPv6 you can still route traffic to and from IPv4 addresses. Please see the [EKS networking best practices](https://aws.github.io/aws-eks-best-practices/networking/index/) for more information.
+
+Using [IPv6 in your cluster](https://docs.aws.amazon.com/eks/latest/userguide/cni-ipv6.html) avoids some of the most common cluster and workload scaling limits. IPv6 avoids IP address exhaustion where pods and nodes cannot be created because no IP address is available. It also has per node performance improvements because pods receive IP addresses faster by reducing the number of ENI attachments per node. You can achieve similar node performance by using [IPv4 prefix mode in the VPC CNI](https://aws.github.io/aws-eks-best-practices/networking/prefix-mode/), but you still need to make sure you have enough IP addresses available in the VPC.
+
+## Limit number of services per namespace
+
+The number of IP tables rules that are created per node with kube-proxy grows with the total number of services in the cluster. Generating thousands of IP tables rules and routing packets through those rules have a performance impact on the nodes and add network latency.
+
+Create Kubernetes namespaces that encompass a single application environment so long as the number of services per namespace is under 500. This will keep service discovery small enough to avoid service discovery limits and can also help you avoid service naming collisions. Applications environments (e.g. dev, test, prod) should use separate EKS clusters instead of namespaces.
```

Comment:
Curious, how you came up with the number "500", since upstream has the threshold has 5000 https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/thresholds.md

Adding couple of lines on reasoning will help to close the disconnect with upstream info.

Reply:
The cluster maximum services is 10000 and most clusters have more than 2 namespaces. 500 was provided from support as a reasonable number per-namespace to also have multiple namespaces with workloads in a real world cluster. Upstream performance testing is idealized without load and updates to CoreDNS, endpoints, etc. are not as heavily considered. I can add additional information to the paragraph if you think it's needed.

Reply:
Your explanation makes total sense. Maybe just adding couple more sentences, with what you just explained, might help readers like me.

Reply:
Definitely agree that production should be a separate cluster. You can probably consolidate the lower environments onto a single cluster, especially if you're cost conscious - and who isn't these days.